src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/Combinations/CombinationForm.razor

```diff
@@ -14,7 +15,10 @@
                Items="_athletes" @bind-Value="_model.Athlete" />
     <NotSelect T="Horse" Label="@Localizer.Get("Horse")" Placeholder="@Localizer.Get("Select a horse")"
                Items="_horses" @bind-Value="_model.Horse" />
-    <RfidTagScanner CombinationNumber="_model.Number" Value="_model.Tag" />
+    @if (StaticOptions.RfidDetection())
```

Comment:
Does `RfidDetection` imply bool value?

Reply:
Still not there. A bool value should propose a question: `IsRfidDetectionEnabled`, `IsUsingRfid` are good suggestions